index.js

```diff
@@ -171,4 +171,26 @@ function setScale(keypath, width, height) {
 
 function setRotation(keypath, degree) {
   rlottieHandler.rlottieModule[0].lottieHandle.setRotation(keypath, degree);
+}
+
+function onResizeSliderDrag(value) {
+  var width = document.getElementById("player").clientWidth;
+  var height = document.getElementById("player").clientHeight;
+  width = width / 4 * 3;
+  height = height / 4 * 3;
+
+  var size = width < height ? width : height;
+  size = size-8;
+  size = size * (value / 100);
+
+  if (size < 20 )
+    size = 20;
```

Comment:
It seems that min size is 20. But why should it be 20 not 30, 40 or other values?

Reply:
There should be some minimum size and I think any value: 10, 16, 20, 24, ... is good enough. I removed unintended space